crates/glaredb/src/local.rs

```diff
@@ -295,14 +304,25 @@ impl LocalSession {
             ("\\max-rows", Some(val)) => self.opts.max_rows = Some(val.parse()?),
             ("\\max-columns", Some(val)) => self.opts.max_rows = Some(val.parse()?),
             ("\\open", Some(path)) => {
-                let new_opts = LocalClientOpts {
-                    data_dir: Some(PathBuf::from(path)),
-                    metastore_addr: None,
-                    ..self.opts.clone()
-                };
-                let new_sess = LocalSession::connect(new_opts).await?;
-                println!("Created new session. New database path: {path}");
-                *self = new_sess;
+                if let Ok(url) = Url::parse(path) {
+                    let new_opts = LocalClientOpts {
+                        data_dir: None,
```

Comment:
I'm assuming this is because we're connecting remotely `cloud_url` and won't have local data to persist? Might make sense to mention it in CLI options

```sh
$ cargo run --bin glaredb -- --help
  -f, --data-dir <DATA_DIR>
          Optional file path for persisting data.

          Catalog data and user data will be stored in this directory.

  -c, --cloud-url <CLOUD_URL>
          URL for connecting to a GlareDB Cloud deployment
```

Reply:
updated